ocaml/typing/typemode.ml

```diff
@@ -92,6 +181,28 @@ let mutable_implied_modalities (mut : Types.mutability) attrs =
     then monadic
     else monadic @ comonadic
 
+let locality_to_regionality : Mode.Locality.Const.t -> Mode.Regionality.Const.t
+    = function
+  | Local -> Local
+  | Global -> Global
+
+let transl_modality ~maturity { txt = Parsetree.Modality modality; loc } =
```

Comment:
This function should probably be moved to its original location (also to minimize diff)